schedule/yast/encryption/cryptlvm+activate_existing+force_recompute_pvm.yaml

```diff
@@ -0,0 +1,40 @@
+---
+description: >
+  Conduct installation activating encrypted partitions, but creating encrypted
+  lvm setup from scratch. Using pre-partitioned disk image to validate encrypted
+  partitions activation and that we can re-ecnrypt the disk.
+name: cryptlvm+activate_existing
+vars:
+  DESKTOP: textmode
```

Comment:
do we need to include this settings for each test suite or it is already set somewhere?

Reply:
It's set in the job group, so I also wondered if we should have it. I guess we can remove it, test suite would work without it and it's not the point.